### PR TITLE
Test yklua in the yk CI.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: .buildbot.sh [hunk 1]
```diff
@@ -117,6 +135,13 @@ for _ in $(seq 10); do
     RUST_TEST_SHUFFLE=1 cargo test
     YKD_NEW_CODEGEN=1 RUST_TEST_SHUFFLE=1 cargo test
 done
+
+# test yklua/hwt in debug mode.
+OLDPATH=${PATH}
```
Comment (ltratt) [inline .buildbot.sh:140]: This is truly horrible :p Can we not just set `PATH=...` in the line below, so it's local to `test_yklua`, just as with the other env vars? Pushing/popping `$PATH` wouldn't be necessary then.

Comment (vext01) [reply .buildbot.sh:140]: I don't know why I didn't think of that.

Comment (vext01) [reply .buildbot.sh]: Try https://github.com/ykjit/yk/pull/1286/commits/706ce13c4f7cf04ff2c2be9886ff0ed5c110cad4

Comment (ltratt) [reply .buildbot.sh]: Everything is obvious in retrospect!

**Comment by ltratt** [comment]
Please squash.

**Comment by ltratt** [comment]
@vext01 Are we still waiting for a squash?

**Comment by vext01** [comment]
squashed.

**Comment by vext01** [comment]
>  not yet implemented: FMul

Maybe there is some non-determinism that causes a different path to be executed.

I can implement `fmul` on my return.

**Comment by vext01** [comment]
Once https://github.com/ykjit/yk/pull/1292 is merged, this one can go in too.

This succeeded with `fmul` implemented, so we should be grand:
```
try_repeat 30 sh -c "YKD_SERIALISE_COMPILATION=0 ../src/lua -e"_U=true" all.lua && YKD_SERIALISE_COMPILATION=1 ../src/lua -e"_U=true" all.lua"
```

**Pull request merged**